python_modules/libraries/dagster-components/dagster_components/scaffold.py

```diff
@@ -15,40 +15,40 @@
 
 
 def scaffold_with(
-    blueprint_cls: Union[type["Blueprint"], "BlueprintUnavailableReason"],
+    scaffolder: Union[type["Scaffolder"], "ScaffolderUnavailableReason"],
 ) -> Callable[[type[T]], type[T]]:
     """A decorator that declares what blueprint is used to scaffold the artifact.
 
     Args:
-        blueprint_ls: A class that inherits from Blueprint
+        blueprint_ls: A class that inherits from Scaffolder
 
     Returns:
         Decorator function that enhances the target class
     """
 
     def decorator(cls: type[T]) -> type[T]:
         # Store the blueprint class as an attribute using the constant
-        setattr(cls, BLUEPRINT_CLS_ATTRIBUTE, blueprint_cls)
+        setattr(cls, BLUEPRINT_CLS_ATTRIBUTE, scaffolder)
         return cls
 
     return decorator
 
 
-def has_blueprint(cls: type) -> bool:
+def has_scaffolder(cls: type) -> bool:
     """Determines if a class has been decorated with blueprint.
```

Comment:
grep for other instances of blueprint so you get comments and local variables too